storage/buffer_test.go

```diff
@@ -78,10 +78,10 @@ func TestSampleRing(t *testing.T) {
 					}
 				}
 				if sold.t >= s.t-c.delta && !found {
-					t.Fatalf("%d: expected sample %d to be in buffer but was not; buffer %v", i, sold.t, buffered)
+					testutil.Fatal(t, "%d: expected sample %d to be in buffer but was not; buffer %v", i, sold.t, buffered)
```

Comment:
This change isn't quite right. The point of the helper is that you move the assertion into the same line. In this case, you should be using `testutil.Assert()`